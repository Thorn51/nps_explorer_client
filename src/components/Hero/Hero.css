/* Colors */
:root {
  --darkGunmetal: #122932;
  --blackCoral: #576066;
  --queenBlue: #386fa4;
  --middleBlue: #80ced7;
  --mintCream: #f7fff7;
  --platinum: #e0e2e3;
}

/* Banner */
.hero {
  height: 40vh;
  border-bottom: solid 2px var(--darkGunmetal);
  background-repeat: no-repeat;
  background-position: center;
  background-size: cover;
}

.title_container {
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100%;
  width: 100%;
  background-color: #12293250;
}

.title {
  height: fit-content;
  color: var(--mintCream);
  font-size: 1.5em;
  text-align: center;
}
