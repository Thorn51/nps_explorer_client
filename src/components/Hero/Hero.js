import React from "react";
import "./Hero.css";

export default function Hero(props) {
  return (
    <div class="hero" role="banner">
      <div class="title_container">
        <h1 class="title">{props.title}</h1>
      </div>
    </div>
  );
}
