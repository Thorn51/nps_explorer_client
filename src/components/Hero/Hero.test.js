import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Hero from "./Hero";

describe("<Hero />", () => {
  it("Renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Hero />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("Renders the UI as expected", () => {
    const tree = renderer.create(<Hero />).toJSON();
    expect("tree").toMatchSnapshot();
  });
});
