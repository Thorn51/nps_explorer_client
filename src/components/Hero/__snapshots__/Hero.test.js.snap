// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> Renders the UI as expected 1`] = `"tree"`;
